runtime/doc/eval.txt

```diff
@@ -1326,6 +1326,14 @@ v:cmdbang	Set like v:cmdarg for a file read/write command.  When a "!"
 		can only be used in autocommands.  For user commands |<bang>|
 		can be used.
 
+				*v:completed_item* *completed_item-variable*
+v:completed_item
+		The selected item in the last candidate. This dictionary
+		contains following items: "word", "abbr", "menu", "kind",
+		"info" (See |complete-items|).  If previous completion is
+		failed or isn't matched, it becomes an empty dictionary.
+		Note: This is for |CompleteDone| event.
```

Comment:
Suggested wording:

> Dictionary containing the most recent |complete-items| after |CompleteDone|. 
> Empty If the completion failed.

Does the empty dictionary look like `{}` or `{"word":"", "abbr":"", ...}`?